tinkerpop3/src/main/java/io/shiftleft/overflowdb/storage/BookKeeper.java

```diff
@@ -24,7 +24,7 @@ protected final long getStartTimeNanos() {
   protected void recordStatistics(long startTimeNanos) {
     totalCount.incrementAndGet();
     totalTimeSpentNanos.addAndGet(System.nanoTime() - startTimeNanos);
-    if (0 == (totalCount.intValue() & 0x0001ffff)) {
+    if (0 == (totalCount.intValue() & 0x0001ffff)) { //131071
```

Comment:
:) I'm sure that means something ... maybe add one more word for posterity?

Reply:
hehe, numbers mean more than a thousand words, ain't that right? ok then